configmap/parse.go

```diff
@@ -45,7 +45,9 @@ func AsString(key string, target *string) ParseFunc {
 func AsBool(key string, target *bool) ParseFunc {
 	return func(data map[string]string) error {
 		if raw, ok := data[key]; ok {
-			*target = strings.EqualFold(raw, "true")
+			val, err := strconv.ParseBool(raw)
+			*target = val // This is basically false always.
```

Comment:
I can't figure out why this comment is true, am I missing something obvious? 😓

Reply:
Eh, bad wording: it is always true if err != nil :)

Reply:
I still dont get it.. doesn't `strconv.ParseBool("true") == true, nil`?

Reply:
you're not looking at the last commit, I think :)

Reply:
ohhh.. I think I get what you mean -- that if err is non-nil, val is false here

Reply:
latest commit is better, but I think I'd go for "If err != nil, val is always false here", but that might just be me, either way I get what you're saying now :)